test/jasmine/tests/cartesian_interact_test.js

```diff
@@ -2421,19 +2421,16 @@ describe('Cartesian plots with css transforms', function() {
             var start = [10, 10];
             var end = [200, 200];
 
-            Plotly.plot(gd, Lib.extendDeep({}, mock))
-            .then(function() {
-                transformPlot(gd, transform);
-                recalculateInverse(gd);
-            })
+            transformPlot(gd, transform);
```

Comment:
Ah I see - you're right, that's better than doing the recalculate function. Somehow it slipped my mind